sdk/ml/azure-ai-ml/tests/sweep_job/unittests/test_sweep_job.py

```diff
@@ -323,3 +323,48 @@ def test_sweep_job_trial_distribution_to_rest(self, distribution) -> None:
         assert sweep_job.inputs == sweep.inputs
         assert sweep_job.outputs == sweep.outputs
         assert sweep_job.identity == sweep.identity
+
+    @pytest.mark.parametrize(
+        "resources",
+        [
+            (JobResourceConfiguration(instance_type="d2", instance_count=2)),
+            (JobResourceConfiguration(instance_type="d2")),
+            (JobResourceConfiguration(instance_count=2)),
+        ],
+    )
+    def test_sweep_job_resources_to_rest(self, resources) -> None:
```

Comment:
this file is for tests involving dsl pipeline. Is it possible to create a dsl pipeline here? you can refer to [a similar test in another PR](https://github.com/Azure/azure-sdk-for-python/pull/30746/files#diff-4394ed09260e0cf4486ec6847ae34daa58b26ddc8b3b1a4936d553763490c2e2). I write this PR to add resources for sweep node in pipeline but seems that it's covered by your PR.

Also, what about make it an e2e test?

Reply:
Thanks, Updated